lib/BUILD

```diff
@@ -98,3 +99,28 @@ bzl_library(
     name = "old_sets",
     srcs = ["old_sets.bzl"],
 )
+
+cc_binary(
+    name = "process_wrapper",
+    srcs = [
+        "process_wrapper/process_wrapper.cc",
```

Comment:
Any reason for not adding a `process_wrapper/BUILD` file instead?

Reply:
I had that originally but I find it cleaner to do this @bazel_skylib//lib:process_wrapper where referencing it, rather than @bazel_skylib//lib/process_wrapper:process_wrapper.
Maybe I should have an alias in //lib ?

Reply:
`@bazel_skylib//lib/process_wrapper:process_wrapper` is equivalent to `@bazel_skylib//lib/process_wrapper` (the default target name is the package's leaf name).

Reply:
I know it's the case in general, but it didn't seem to work in this context:
"_process_wrapper": attr.label(
            default = "@bazel_skylib//lib:process_wrapper",
            executable = True,
            allow_single_file = True,
            cfg = "exec",
        ),

Reply:
If that's the code snippet you tried and failed... the `:` is wrong in the `default` value. Should be a slash.

Reply:
Had a slash in there I think, but my memory might fail me. Let me try again!

Reply:
Fixed